lib/pi_piper/bcm2835.rb

```diff
@@ -41,6 +42,21 @@ def self.pin_read(pin)
       File.read("/sys/class/gpio/gpio#{pin}/value").to_i
     end
 
+    # Exports pin and subsequently locks it from outside access
+    def self.lock_pin(pin)
```

Comment:
I'll have called this methods #export #unexport for sake of readibility
maybe we can add a #exported? (see comment below) 

but anyhow this works great.